hermes-guide-days-website/src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,141 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+
+.topnav {
+    overflow: hidden;
+    background-color: #cc3c3b;
+  }
+
+.topnav .navbar-item {
+  float: left;
+  width:210px;
+  margin: 3px 5px 3px 5px;
+  display: block;
+
+}
+.topnav .navbar-item img {
+  width:100%;
+  height:100%;
+}
+
+  .topnav a {
+    float: right;
+    display: block;
+    color: #e5e5e5;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+    font-family: Montserrat;
```

Comment:
you won't need to declare the font-family separated as it will be applied globally